test/app.e2e-spec.ts

```
@@ -354,18 +354,29 @@ describe('NestJS Starter E2E Tests', () => {
+            );
+          });
+      }
+
+      // The 4th request should be rate limited
+      const response = await request(app.getHttpServer())
+        .get('/test/rate-limit')
+        .expect(429);
+
+      // Verify the presence of the Retry-After header
+      expect(response.headers).toHaveProperty('retry-after');
+      expect(parseInt(response.headers['retry-after'], 10)).toBeGreaterThan(0);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)